packages/data-service/src/instance-detail-helper.spec.ts

```diff
@@ -241,6 +245,53 @@ describe('instance-detail-helper', function () {
           'documentdb'
         );
       });
+      it(`should be identified as atlas with hostname correct hostnames`, function () {
+        ['myserver.mongodb.net', 'myserver.mongodb-dev.net'].map(
+          async (hostname) => {
+            const client = createMongoClientMock({
+              hosts: [{ host: hostname, port: 9999 }],
+              commands: {
+                buildInfo: {},
+                getCmdLineOpts: fixtures.DOCUMENTDB_CMD_LINE_OPTS,
```

Comment:
should not affect the test, but we could use `CMD_LINE_OPTS` to avoid confusion.